wallet-api/src/Wallet/Generators.hs

```diff
@@ -50,6 +65,34 @@ import           Ledger
 import qualified Wallet.API      as W
 import           Wallet.Emulator as Emulator
 
+-- $wallets
+-- 'wallet1', 'wallet2', ... 'wallet10' are ten predefined 'Wallet' values.
+--
+-- The private keys can be found in the 'sign.input' file linked from
+-- http://ed25519.cr.yp.to/software.html.
+
+wallet1, wallet2, wallet3, wallet4, wallet5, wallet6, wallet7, wallet8, wallet9, wallet10 :: Wallet
+wallet1 = Wallet $ fromHex "9d61b19deffd5a60ba844af492ec2cc44449c5697b326919703bac031cae7f60"
+wallet2 = Wallet $ fromHex "4ccd089b28ff96da9db6c346ec114e0f5b8a319f35aba624da8cf6ed4fb8a6fb"
+wallet3 = Wallet $ fromHex "c5aa8df43f9f837bedb7442f31dcb7b166d38535076f094b85ce3a2e0b4458f7"
+wallet4 = Wallet $ fromHex "691865bfc82a1e4b574eecde4c7519093faf0cf867380234e3664645c61c5f79"
+wallet5 = Wallet $ fromHex "3b26516fb3dc88eb181b9ed73f0bcd52bcd6b4c788e4bcaf46057fd078bee073"
+wallet6 = Wallet $ fromHex "edc6f5fbdd1cee4d101c063530a30490b221be68c036f5b07d0f953b745df192"
+wallet7 = Wallet $ fromHex "a980f892db13c99a3e8971e965b2ff3d41eafd54093bc9f34d1fd22d84115bb6"
+wallet8 = Wallet $ fromHex "9acad959d216212d789a119252ebfe0c96512a23c73bd9f3b202292d6916a738"
+wallet9 = Wallet $ fromHex "d5aeee41eeb0e9d1bf8337f939587ebe296161e6bf5209f591ec939e1440c300"
+wallet10 = Wallet $ fromHex "0a47d10452ae2febec518a1c7c362890c3fc1a49d34b03b6467d35c904a8362d"
+
+-- | A list of 10 wallets. 
+--   TODO: Generate wallets from random private keys (I couldn't find a way to 
+--         do this in 'Crypto.ECC.Ed25519Donna' in 'cardano-crypto') 
```

Comment:
Have we opened an issue?

Reply:
There's this issue: https://github.com/input-output-hk/cardano-crypto/issues/37 but I'll open one specifically about generating random keys, which you [can do](https://hackage.haskell.org/package/cryptonite-0.25/docs/Crypto-PubKey-Ed25519.html#v:generateSecretKey) in `cryptonite`

Reply:
https://github.com/input-output-hk/cardano-crypto/issues/57